src/Domain/CoordinateMaps/Frustum.cpp

```diff
@@ -538,7 +593,10 @@ bool operator==(const Frustum& lhs, const Frustum& rhs) {
          lhs.phi_ == rhs.phi_ and
          lhs.half_opening_angle_ == rhs.half_opening_angle_ and
          lhs.one_over_tan_half_opening_angle_ ==
-             rhs.one_over_tan_half_opening_angle_;
+             rhs.one_over_tan_half_opening_angle_ and
+         lhs.inner_radius_ == rhs.inner_radius_ and
+         lhs.radius_ == rhs.radius_ and
+         lhs.zeta_distribution_ == rhs.zeta_distribution_;
```

Comment:
duplicate I think, also radius

Probably best to order these the same as the order of attributes in the class, if they aren't already

Reply:
radius is still duplicate